schedule/sle-perf/hanaperf_installation.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: hanaperf_installation
+description: >
+  OS installation for HANA performance
+vars:
+  MULTIPATH: '1'
+  MULTIPATH_CONFIRM: 'yes'
+  SEPARATE_HOME: '1'
+  NUMDISKS: '7'
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - '{{scc_registration}}'
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_smalldisk_storageng
+  - '{{separate_home}}'
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/install_qatestset
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/full_run
+conditional_schedule:
+  scc_registration:
+    SCC_REGISTER:
+      none:
+        - '{{multipath}}'
+        - installation/scc_registration
+      installation:
+        - installation/scc_registration
+        - '{{multipath}}'
+  multipath:
```

Comment:
Similar to `separate_home`. As `MULTIPATH=1` always (it is set on the 6th line), then this check for `MULTIPATH: 1` is not needed.

Reply:
Remove the values in `vars`, and will define by command line